dimasacat_andronicus/dimasacat_andronicus.html

```diff
@@ -0,0 +1,247 @@
+            happy, sad, or motivated, there's a song for every moment.
+          </p>
+        </div>
+      </ul>
+    </div>
+    <div class="container-works">
+      <h2>My Works</h2>
+      <ol class="list-works">
+        <div class="works-style">
+          <li><a href="#">Hello World in C++</a></li>
+        </div>
+        <div class="works-style">
+          <li>
+            <a
+              href="#"
+              >Photo Editor Design in Figma</a
+            >
+          </li>
+        </div>
+        <div class="works-style">
+          <li><a href="./countries/country_index.html">Countries of the World</a></li>
+        </div>
+        <div class="works-style">
+          <li>
+            <a
+              href="#"
+              >Google Site Portfolio</a
+            >
+          </li>
+        </div>
+        <div class="works-style">
+          <li><a href="#">If Else in Python</a></li>
+        </div>
+      </ol>
+    </div>
+    <div class="container-testimonials">
+      <h2>Testimonials</h2>
+      <div class="guide-testimonials">
+        <div class="testimonials-style">
+          <p>
+            Wow Andronicus. Welcome to the team. Thank you for your 
+            unwavering dedication to our shared goals. You make every day 
+            at work a rewarding journey. XD
+          </p>
+          <p>- Christian Harrel Y. Go</p>
+        </div>
+        <div class="testimonials-style">
+          <p>
+            "I am delighted to provide a testimonial for my incredible 
+            teammate, Andronicus has been an absolute pleasure and has 
+            significantly contributed to the success of our team."
+          </p>
+          <p>- Mark Ryan J. Odrunia</p>
+        </div>
+        <div class="testimonials-style">
+          <p>
+            "Andro excels in everything, an extremely competent person 
+            with a very deep passion with everything that he does"
+          </p>
+          <p>- Jhean Galope</p>
+        </div>
+        <div class="testimonials-style">
+          <p>
+            "Andro thank you for helping me on some coding session and 
+            lesson. Always remember you have us THE VRZN team always 
+            got you."
+          </p>
+          <p>- Adrian Naoe</p>
+        </div>
+      </div>
+    </div>
+    <div class="container-gallery">
+      <h2>Gallery</h2>
+      <div class="guide-gallery">
+        <img
+          class="image-gallery"
+          src="./images/dain.png"
+          alt="Dainsleif Image from Genshin Impact"
+        />
+        <img
+          class="image-gallery"
+          src="./images/mclarenp1.jpg"
+          alt="A Mclaren P1 Sports Car Image "
+        />
+        <img
+          class="image-gallery"
+          src="./images/paris.jpg"
+          alt="Eiffel Tower Image"
+        />
+        <img
+          class="image-gallery"
+          src="./images/landscape.jpg"
+          alt="Beautiful Landscape Image"
+        />
```

Comment:
refer to this [Guideline](https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g29e6ce31160_0_27)